packages/server/providers.yaml

```diff
@@ -189,6 +189,13 @@ linkhut:
     auth_mode: OAUTH2
     authorization_url: https://ln.ht/_/oauth/authorize
     token_url: https://api.ln.ht/v1/oauth/token
+mailchimp:
+    auth_mode: OAUTH2
+    authorization_url: https://login.mailchimp.com/oauth2/authorize
+    token_url: https://login.mailchimp.com/oauth2/token
+    disable_scope: true
```

Comment:
To make sure I understand this: Mailchimp doesn't require scopes at all? In this case, can users just not configure any scope in their Provider Config? So that we avoid another param in the template?

Reply:
Yes,  I thought of this and I think it is a better approach. I am happy to do this. I only thought of a good approach. If we follow this path, then we will need to make **_oauth_scopes_** in **_nango_configs_** nullable. Then also update the UI. In any case, I think I will just create a separate issue and PR for this.  What do you think @bastienbeurier.

Reply:
Yes, that'd be great! Thanks :)